test/tracker-component.test.js

```diff
@@ -9,6 +9,24 @@ import { getUserId, getCartId } from '../src/lib/local-storage';
 
 const { sevenDays, storage } = constants;
 
+const sampleItem = [
```

Comment:
`sampleItem` could just be an object, so you don't have to do `sampleItem[0]`. With arrays, the variable should be `sampleList`.

Reply:
ah true, will change!